game-core/src/main/java/games/strategy/triplea/image/MissingImageException.java

```diff
@@ -0,0 +1,14 @@
+package games.strategy.triplea.image;
+
+import games.strategy.triplea.image.UnitImageFactory.ImageKey;
+import games.strategy.triplea.util.UnitCategory;
+
+public class MissingImageException extends RuntimeException {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5910459)